axelrod/strategies/gobymajority.py

```diff
@@ -51,14 +51,16 @@ def __init__(self, memory_depth: int = float('inf'), soft: bool = True) -> None:
             self.memory = self.classifier['memory_depth']
         else:
             self.memory = 0
-
         self.name = (
             'Go By Majority' + (self.memory > 0) * (": %i" % self.memory))
         if self.soft:
             self.name = "Soft " + self.name
         else:
             self.name = "Hard " + self.name
 
+    def __repr__(self):
```

Comment:
This is the default `__repr__` from `Player`, yes? If so I think we don't need to add it again explicitly.